dependencies/configs/besu.go

```diff
@@ -0,0 +1,14 @@
+package configs
+
+var BesuConfigDependencies = map[string]ClientConfigDependency{
+	besuMainnetConfigName: &clientConfig{
+		url:      "https://raw.githubusercontent.com/lukso-network/network-configs/main/mainnet/besu/besu.yaml",
+		name:     besuMainnetConfigName,
+		filePath: MainnetConfig + BesuYamlPath,
+	},
+	besuTestnetConfigName: &clientConfig{
+		url:      "https://raw.githubusercontent.com/lukso-network/network-configs/main/mainnet/besu/besu.yaml",
```

Comment:
Please change to `testnet` before merge @Wolmin 

It's `.../network-configs/main/mainnet/besu...` now.